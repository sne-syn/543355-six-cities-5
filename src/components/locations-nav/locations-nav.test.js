import React from 'react';
import renderer from 'react-test-renderer';
import LocationsNav from './locations-nav';

jest.mock(`../locations-item/locations-item`, () => `LocationsItem`);

test(`Render LocationsNav`, () => {
  const tree = renderer
    .create(<LocationsNav tab={true} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
