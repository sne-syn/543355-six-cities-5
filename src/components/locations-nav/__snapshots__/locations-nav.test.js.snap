// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render LocationsNav 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <LocationsItem
      cityName="Paris"
      tab={true}
    />
  </li>
  <li
    className="locations__item"
  >
    <LocationsItem
      cityName="Cologne"
      tab={true}
    />
  </li>
  <li
    className="locations__item"
  >
    <LocationsItem
      cityName="Brussels"
      tab={true}
    />
  </li>
  <li
    className="locations__item"
  >
    <LocationsItem
      cityName="Amsterdam"
      tab={true}
    />
  </li>
  <li
    className="locations__item"
  >
    <LocationsItem
      cityName="Hamburg"
      tab={true}
    />
  </li>
  <li
    className="locations__item"
  >
    <LocationsItem
      cityName="Dusseldorf"
      tab={true}
    />
  </li>
</ul>
`;
